sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -718,6 +718,11 @@ public partial class BlobServiceStatistics
         internal BlobServiceStatistics() { }
         public Azure.Storage.Blobs.Models.BlobGeoReplication GeoReplication { get { throw null; } }
     }
+    public enum BlobSetExpiryRelativeTo
+    {
+        BlobCreationTime = 0,
```

Comment:
Is the `Blob` prefix is needed here?  It feels redundant given the type name and API it'll be used on.

Reply:
What if Set Expiry is added to SMB Files or Queues?

Reply:
I'm talking specifically on `CreationTime`, not the enum name.  The github UI doesn't make that super clear.

Reply:
Oh man bad, ya we can change it.

Reply:
Changed to ```Now``` in both Blobs and Data Lake.